LiskHQ/lisk-desktop
# Hardware wallet account management

### What was the problem?

This PR resolves #5237

### How was it solved?

Adding a separate modal to import hardware account from

### How was it tested?

1. `git pull`, `yarn build`, `yarn dev`, `LISK_DESKTOP_URL="http://localhost:8080" DEBUG=true yarn run start`
2. Connect HW and enter pin
3. Click "Import Account" from the Hw Status icon in the topbar
4. **Expected**: Show a modal asking user to open the LiskApp
5. Open Lisk App
6. **Expected**: 
 1 Show Loading state 
 2 Show 3 Accounts
7. Click Load More
8. **Expected**: 
 1 Show Loading state 
 2 Show 6 Accounts
9. Click on 2nd account to import it
10. **Expected**: 
 1 Modal closes 
 2  Imported account becomes current account
11. Rename the HW account to "Test" and Click "Import Account" from the Hw Status icon in the topbar
12. **Expected**:   
 1 Show Loading state
 2 Show 3 Accounts where the second account is named "Test"
13. Close modal and Try to do a send transaction
14. **Expected**: It should work to send a transaction
